app/controllers/concerns/form_answer_mixin.rb

```diff
@@ -110,7 +110,7 @@ def save_or_load_search!
 
       unless @processor.valid?
         flash[:bulk_error] = @processor.base_error_messages
-        redirect_to admin_form_answers_path(year: params[:year], search_id: @processor.search_id) and return
+        redirect_to admin_form_answers_path(year: params[:year], search_id: @processor.search_id) + "#bulk_error" and return
```

Comment:
you can probably use `anchor` keyword.
```suggestion
        redirect_to admin_form_answers_path(year: params[:year], search_id: @processor.search_id, anchor: "bulk_error") and return
```